package-lock.json

```diff
@@ -7758,6 +7758,12 @@
             "path-is-absolute": "^1.0.0",
             "rimraf": "^2.2.8"
           }
+        },
```

Comment:
It's still in the package lock @salomao-rodrigues :P

Reply:
I think that's a dependency of a dependency of storybook.
I got it removed in my last commit:
https://github.com/uswitch/trustyle/pull/3/commits/26ada1860b43d0746d28cb6d8f1e65029de1db63

Reply:
haha